### PR TITLE
An embedded explicit Runge-Kutta integrator

### PR DESCRIPTION
**Pull request by pleroy**
🍐 for #3358.

### CODE REVIEW

File: integrators/embedded_explicit_runge_kutta_integrator_body.hpp [hunk 1]
```diff
@@ -0,0 +1,417 @@
...
+
+    if (first_same_as_last) {
+      last_f = f;
+    }
+
+    // Increment the solution with the high-order approximation.
+    t.Increment(h);
+    for_all_of(Δŷ, ŷ).loop([](auto const& Δŷ, auto& ŷ) {
+      int const dimension = ŷ.size();
+      for (int k = 0; k < dimension; ++k) {
```
Comment (eggrobin) [inline integrators/embedded_explicit_runge_kutta_integrator_body.hpp:271]: Rename this index to `l`.

File: integrators/embedded_explicit_runge_kutta_integrator_body.hpp [hunk 2]
```diff
@@ -0,0 +1,417 @@
...
+using quantities::Quotient;
+
+template<typename Method, typename... StateElements>
+EmbeddedExplicitRungeKuttaIntegrator<Method, StateElements...>::
+EmbeddedExplicitRungeKuttaIntegrator() {
+  // The first node is always 0 in an explicit method.
+  CHECK_EQ(0.0, c_[0]);
+  if (first_same_as_last) {
+    // Check that the conditions for the FSAL property are satisfied, see for
+    // instance [DEP87a], equation 3.1.
```
Comment (eggrobin) [inline integrators/embedded_explicit_runge_kutta_integrator_body.hpp:37]: Do we have a citation more specific to RK rather than RKN integrators?

**Comment by pleroy** [comment]
retest this please

**Review by eggrobin — APPROVED** [review]

**Comment by pleroy** [comment]
retest this please

**Pull request merged**